tests/acceptance/features/webUIFilesDetails/fileDetails.feature

```diff
@@ -62,21 +62,21 @@ Feature: User can open the details panel for any file or folder
     When the user switches to "links" accordion item in details panel using the webUI
     Then the "links" details panel should be visible
 
-  @skip @yetToImplement @comments-app-required @public_link_share-feature-required
+  @comments-app-required @public_link_share-feature-required
   Scenario: user shares a file through public link and then the details dialog should work in a Shared by link page
     Given user "Alice" has created folder "simple-folder"
     And the user has browsed to the files page
-    And the user has created a new public link for folder "simple-folder" using the webUI
+    Given user "Alice" has shared folder "simple-folder" with link with "read" permissions
     When the user browses to the shared-by-link page
     Then folder "simple-folder" should be listed on the webUI
```

Comment:
I think this need not be checked in this test. what do you think @Talank  @individual-it ?

Reply:
which check?

Reply:
Then folder "simple-folder" should be listed on the webUI
 this one